src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -253,13 +253,10 @@ private void setUpKeyBindings() {
         listView.addEventFilter(KeyEvent.KEY_PRESSED, event -> {
             Optional<KeyBinding> keyBinding = preferences.getKeyBindingRepository().mapToKeyBinding(event);
             if (keyBinding.isPresent()) {
-                switch (keyBinding.get()) {
-                    case DELETE_ENTRY:
-                        deleteAttachedFilesWithConfirmation();
-                        event.consume();
-                        break;
-                    default:
-                        // Pass other keys to children
+                // Pass other keys to children
+                if (keyBinding.get() == KeyBinding.DELETE_ENTRY) {
+                    deleteAttachedFilesWithConfirmation();
+                    event.consume();
```

Comment:
I know, IntelliJ complains that this should be an if clause, but...
Everywhere a keybinding test has more than one case its a switch, so for consistency reasons I would propose to keep this as a switch, even though it got only one case.